src/seedu/addressbook/data/person/Address.java

```diff
@@ -9,11 +9,17 @@
 public class Address {
```

Comment:
Avoid accumulating unrelated changes into a single PR. 
Tip: If you are currently on `branch1` and want to create a new `branch2`, you need to switch back to `master` branch before creating `branch2`. If not, changes in `branch1` will appear in the PR created from `branch2` later.